packages/components/src/Components/InsightsLabel/InsightsLabel.js

```diff
@@ -0,0 +1,55 @@
+import './labels.scss';
+
+import React, { useEffect, useState } from 'react';
+
+import { AngleDoubleDownIcon } from '@patternfly/react-icons';
+import { AngleDoubleUpIcon } from '@patternfly/react-icons';
+import CriticalIcon from './CriticalIcon';
+import { EqualsIcon } from '@patternfly/react-icons';
+import { Label } from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+
+const VALUE_TO_STATE = {
+    1: { icon: <AngleDoubleDownIcon/>, text: 'Low' },
+    2: { icon: <EqualsIcon/>, text: 'Moderate' },
+    3: { icon: <AngleDoubleUpIcon/>, text: 'High' },
+    4: { icon: <CriticalIcon/>, text: 'Critical' }
+};
+
+const InsightsLabel = ({ value, text, hideIcon, labelProps }) => {
```

Comment:
I think labelProps can be replaced by using `...rest` and `{...rest}` as it is the only component so I don’t see any reason to specify it.